creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -110,6 +121,8 @@ const _syncRouteController = async (req, res) => {
    * Else, debounce + add sync to queue
    */
   const data = generateDataForSignatureRecovery(req.body)
+  const syncUuid = uuid()
+  await setSyncStatus(syncUuid, 'waiting', syncType)
```

Comment:
it's hard bc its type is a concatenation of strings rather than a key-value map BUT once this file is typed it'll make compile errors if you put in any value that's outside the type (ex: "waitinggg" as a typo would make an error)